interface/elements/msc-simple-login-view/msc-simple-login-view.js

```diff
@@ -1,6 +1,12 @@
 var fs = require('fs');
 var ngui = require('nw.gui');
 var nwin = ngui.Window.get();
+
+var timesync = require('timesync');
+var ts = timesync.create({
+  server: 'http://time.is',  // either a single server,
```

Comment:
This mistake is one me, seems like they don't have an api. Let's change it to `time.google.com`